### DISCUSSION TITLE
Exists bar menu for linux port?

### DISCUSSION THREAD

**Discussion by ThingUroboros** [86Box/86Box#1998] [General]
Exists bar menu for linux port?

**Comment by dhrdlicka** [comment]
We are working on a Qt port (#2016) which adds a proper GUI to the Linux build.

**Reply by dob205** [reply]
And since yesterday evening it's actually merged into the master.

**Reply by jriwanek** [reply]
Released: https://github.com/86Box/86Box/releases/tag/v3.2.1

**Comment by jriwanek** [comment]
Closing as discussion resolved
